test/createStore.spec.js

```diff
@@ -548,4 +548,12 @@ describe('createStore', () => {
       store.replaceReducer(() => {})
     ).toNotThrow()
   })
+
+  it('throws if listener is not a function', () => {
+    const store = createStore(reducers.todos)
+
+    expect(() =>
+      store.subscribe()
+    ).toThrow()
```

Comment:
Added a few extra checks for the types mentioned. I've gone with the assertion style I've seen elsewhere rather than something like:

```
const cases = [undefined, null, ''];
    cases.map(arg =>
      expect(() =>
        store.subscribe(arg))
      .toThrow())
```